### ISSUE TITLE
[Feature Request] Clear clipboard after 15 s

### ISSUE THREAD

**Issue by lcworld** [nuxsmin/sysPass#833]
After clicking on "Copy password to clipboard", the password stays in it.
It would be interresting to clear the password after an amount of time.

The amount of time could be a global settings.
The clear of clipboard should only be done if it still contains the password.
A popup with the remaing time could be a great addition.

**Comment by vmario89** [comment]
good idea. this is a standard feature in most password stores, like KeePass

**Comment by nuxsmin** [comment]
Sure! It could be easily implemented

**Comment by bitking** [comment]
Hi, has this been implemented in 3.1?

cant seem to find the feature.
